metadoc-js/src/main/scala/metadoc/Metadoc.scala

```diff
@@ -20,7 +20,11 @@ object MetadocApp extends js.JSApp {
       indexBytes <- fetchBytes("metadoc.index")
       index = Index.parseFrom(indexBytes)
       bytes <- fetchBytes(
-        "semanticdb/" + index.files(0).replace(".scala", ".semanticdb")
+        "semanticdb/" +
+          index.files
+            .find(_.endsWith("Doc.scala"))
+            .get
+            .replace(".scala", ".semanticdb")
```

Comment:
I think it makes sense to start with an empty page + instructions on opening files

![screen shot 2017-07-01 at 18 27 46](https://user-images.githubusercontent.com/1408093/27763805-0b9df070-5e8b-11e7-82d9-14a4606a2d59.png)